src/cli/domain/package-template.js

```diff
@@ -20,11 +20,7 @@ var compileView = function(viewPath, type) {
         preCompiledView;
 
   if(type === 'jade'){
-    preCompiledView = jade.compileClient(template, {
-      filename: viewPath,
-      compileDebug: false,
-      name: 't'
-    }).toString().replace('function t(locals) {', 'function(locals){');
+    preCompiledView = jade.precompile(template, { filename: viewPath });
```

Comment:
I think I would like to do something similar to the oc-client here rather than all the ifs. Something like

```js
var precompilers = { jade, handlebars};
// and then
if(!precompilers[type]){
  throw strings.errors.cli.TEMPLATE_TYPE_NOT_VALID;
}
var preCompiledView = precompilers[type].precompile(template, { viewPath });
```